features/netsocket/emac-drivers/TARGET_Cypress/TARGET_PSOC6/TARGET_WHD/interface/WhdSTAInterface.h

```diff
@@ -0,0 +1,117 @@
+/* WHD implementation of NetworkInterfaceAPI
+ * Copyright (c) 2017 ARM Limited
```

Comment:
Please add SPDX identifier to new files (also the year 2017 is correct here?)

Reply:
Fixed by https://github.com/ARMmbed/mbed-os/pull/10692/commits/c115f23028c674329e859c26f4b7ad06f5d3d66c#diff-8d3ee587e2e526b518f9d8b2995fd36cR3